aspnet/mvc/models/file-uploads.rst

```diff
@@ -0,0 +1,197 @@
+File Uploads
+============
+
+By `Steve Smith`_
+
+ASP.NET MVC actions support uploading of one or more files, using simple model binding for smaller files, or streaming for larger files.
+
+.. contents:: Sections
+  :local:
+  :depth: 1
+
+`View or download sample from GitHub <https://github.com/aspnet/Docs/tree/master/aspnet/mvc/models/file-uploads/sample>`_.
+
+Uploading Small Files with Model Binding
+----------------------------------------
+
+To upload small files, you can use a multi-part HTML form or construct a POST request using JavaScript. An example form using Razor which supports multiple uploaded files is shown below:
+
+.. literalinclude:: file-uploads/sample/FileUploadSample/Views/Home/Index.cshtml
+  :language: html
+  :lines: 7-20
+  :dedent: 8
+
+
+In order to support file uploads, HTML forms must specify an ``enctype`` of ``multipart/form-data``. The ``files`` input element shown abovesupports uploading multiple files; omit the ``multiple`` attribute on this input element to allow just a single file to be uploaded. The above markup renders in a browser as:
```

Comment:
You can do a lot with styling the upload HTML - see for instance http://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/

However that seemed out of scope for the doc, which is focused on the server side of things, primarily.